### ISSUE TITLE
Programs apple venn diagram graphic appears tiny on Internet Explorer

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#860]
There is a bug on the Internet Explorer browser where the programs' apple venn diagram graphic appears too small. Our partner sent this photo:

![image](https://user-images.githubusercontent.com/5550969/28948210-d7711d14-7879-11e7-820d-5d9b9e7689ff.png)

**Comment by theandrewbriggs** [comment]
I don't have a great way of testing this, @smarziano can you verify this still looks like this? There were a lot of style changes in/around the Chalk font.

**Comment by smarziano** [comment]
Yes, @theandrewbriggs This is still a problem on Internet Explorer.

**Comment by theandrewbriggs** [comment]
@smarziano @amynicole I'm officially stumped on this one. Amy, can you export this as a PNG?
